upup/pkg/fi/nodeup/nodetasks/service.go

```diff
@@ -87,22 +86,6 @@ func (s *Service) String() string {
 	return fmt.Sprintf("Service: %s", s.Name)
 }
 
-func NewService(name string, contents string, meta string) (fi.Task, error) {
```

Comment:
Could you leave this one in?
I am using this one in #9137

Reply:
Where? I don't see that PR adding a file to the model directory. If you're creating a `nodetasks.Service` you should just use a literal, not call `NewService` with a string.